docs/adr/adr-012-daser-parallelization.md

```diff
@@ -0,0 +1,78 @@
+# ADR #012: DASer parallelization
+
+## Changelog
+
+- 2022-9-12: Started
+
+## Authors
+
+@walldiss
+
+## Context
+
+DAS is the process of verifying the availability of
+block data by sampling chunks or shares of those blocks. The `das` package implements an engine to continuously ensure the availability of the chain's block data via the `Availability` interface.
```

Comment:
```suggestion
block data by sampling chunks or shares of those blocks. The `das` package implements an engine to ensure the availability of the chain's block data via the `Availability` interface.
```

`continuously` sounds like we request the same height frequently over some period of time